discovery/community.py

```diff
@@ -218,6 +218,11 @@ def on_results(success):
 
         super(DiscoveryCommunity, self).initialize()
 
+    def unload_community(self):
+        super(DiscoveryCommunity, self).unload_community()
+        if self.bootstrap:
+            self.bootstrap.stop()
```

Comment:
I think it would be nicer if  `Bootstrap` extended `TaskManager` and took care of it's own stuff instead of doing it @ 212:214 and here. A `start()` method should be implemented to complement the `stop()` we already have.